src/clj_gcp/storage/core.clj

```diff
@@ -13,6 +13,7 @@
 
 (defprotocol StorageClient
   (get-blob [this bucket-name blob-name])
+  (exists [this bucket-name blob-name])
```

Comment:
In Clojure the naming convention is to have a question mark at the end of the function name if that function is a predicate

Reply:
So this would be `exists?`